.freeCodeCamp/tooling/test-utils.js

```diff
@@ -101,6 +101,14 @@ async function getDirectory(path) {
   return files;
 }
 
+/**
+ * Create a directory
+ * @param {string} path Path relative to root of working directory
+ */
+async function makeDirectory(path) {
+  await mkdir(join(ROOT, path));
+}
+
```

Comment:
I am happy to add this, but we might be better off adding `ROOT` to the exports, and declaring `fs/promises` as global in `test.js`.

That is, this function does very little.

Reply:
You're probably right. Lets just go with it like this for now. There's a few other functions in there (probably from me) that don't really do much either. Maybe we can go back and clean them all up at the same time.